src/garage/torch/distributions/tanh_normal.py

```diff
@@ -0,0 +1,154 @@
+"""A Gaussian distribution with tanh transformation."""
+from collections import namedtuple
+
+import torch
+from torch.distributions import Distribution, MultivariateNormal
```

Comment:
use Normal, not MultivariateNormal unless your covariance matrix is non-diagonal